walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,87 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors; 
+import java.util.stream.Stream; 
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    int dayStart = TimeRange.START_OF_DAY;
+    int dayEnd = TimeRange.END_OF_DAY;
+    int start = dayStart;
+    int end;
```

Comment:
Can we make the names "start" and "end" somewhat more descriptive? Are these the start and end of individuals' availability?

Reply:
Sure thing, they're the start and end times of each event that the individuals have scheduled, I use the end of the previous event and the start of the next to determine availability in between.

Reply:
I've changed them to "startEvent" and "endEvent" reflecting that they are the start time and the end time of the events.